README.markdown

```diff
@@ -428,6 +426,10 @@ Generated code will be placed in the Gradle build directory.
 
   Note that `addGrpcMetadata`, `addNestjsRestParameter` and `returnObservable` will still be false.
 
+- With `--ts_proto_opt=http=true`, the defaults will change to generate http friendly types & service metadata that can be used to create your own http client. See the [http readme](HTTP.markdown) for more information and implementation examples.
```

Comment:
Can we call this `grpcHttp` or `googleApiHttp`? Hm, or maybe the existing config option could have `outputClientImpl=google-api-http`? I think I like that option the most... Does that seem fine, adding a new value to `outputClientImpl`?

Granted, its a slight misnomer b/c we're not generating the true "client impl", but instead some config to be used by the client impl...but, dunno, that seems fine?